client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/customs-step/index.js

```diff
@@ -39,7 +43,16 @@ const CustomsStep = ( props ) => {
 			toggleStep={ props.toggleStep }
 			isSuccess={ isSubmitted && ! hasNonEmptyLeaves( errors ) }
 			isError={ isSubmitted && hasNonEmptyLeaves( errors ) } >
-			<Checkbox checked={ Boolean( dummyFieldChecked ) } onChange={ props.toggleDummyField } /> Check this to make validation pass.
+			{ Object.keys( packageDescriptions ).map( ( packageId, index ) =>
+				<div className="customs-step__package-container" key={ packageId }>
+					{ index ? <hr /> : null }
+					<p className="customs-step__package-name">{ packageDescriptions[ packageId ] }</p>
```

Comment:
This isn't a blocker for this PR at all, but something that's been on my mind in relation to the data structures we use. I wanted to see what you thought of potentially changing `packageDescriptions` to be an array of packages like this:
```
[ { 
   id: 'package_id',
   description: 'package description',
 } ]
```

It would make mapping more intuitive to read, and easier to write. Doesn't have to happen here, but depending on your comments we could keep it in mind for next time.

Reply:
As a counterpoint to what I said - I just read through the Redux docs again recently, and their example for a normalized state in fact uses a similar data structure to what we already have:

```
[ 
  id1: { 
     id: 'id1',
     description: 'package description',
   } ,
  id2: {},
  // etc ...
]
```

Makes it easier to update the state. But, they do add the `id` in the object as well (as seen above) - which makes it easier to map over values 👌